_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe4.PNG' | relative_url }}" />
+    </div>
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe5.PNG' | relative_url }}" />
+    </div>
+  </div>
+</figure>
+
+In both the cases, there was a common cause between the quantities we were comparing, and due to this common cause, the relationship of the quantities can be very arbitrary. So, we can't rely on them for any kind of recommendation. We should fix the value of the common cause, and then see the relation between the two quantities we want.
+
+### Strange Dependence
+Example - If we go back 50 years, we observe that female college students are smarter than male student on average. 
+Gender and IQ are two very different quantities, and we can safely say that they are independent. But given a common cause, like say going to college, now they become dependent. A simple way to understand this is as follows. Consider two independent variable X and Y. But if we are given the sum of X and Y, now they both are dependent.
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe6.PNG' | relative_url }}" />
+    </div>
+  </div>
+</figure>
+
+#### Monty Hall problem
+The setup is as follows. There are 3 doors, behind one of which there is 1000\$. If we select that door, we get the prize money. So we make our initial guess, then the game show host opens one of the empty doors and shows there is no money behind that door. Now should we stick with our initial choice or switch to the other door.
+
+The answer is to switch our choice. If we switch, our probabilty of winning becomes 2/3 rather than the 1/3 if we stick to our choice. We can think of it as follows. There are three options to select from initially, two empty doors and one with the prize money. If we are always switching, we will end up with the prize money if we have initially chosen the empty door and we will lose if we have initially selected the door with the prize money. So to win, we need to select the empty door at first whose probability is 2/3. So the chance of us winning the car if we switch is 2/3.
+
+### Causal Models
+If we have causal models, it is very easy to do intervention. We just need to delete all the incoming edges into the nodes which we are intervening and then do inference on the obtained graphical structure.
+
+Also if we know the causal structure, we can apply some of the known distributions into new scenarios. This idea is used in transfer learning.
+
+### Prediction
```

Comment:
I'm very confused by these headers. Should "Prediction", "Intervention", "Counterfactual Thinking" headers be . under "Causal Models"? If so, please change "### Prediction" to "#### Prediction", and so on.